### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by emalihin**

### CODE REVIEW

File: templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{{- $root := . }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.name }}
+  namespace: dev
```
Comment (adamstrawson) [inline templates/deployment.yaml:6]: Need to remove `namespace`

File: templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            - name: config-volume
+              mountPath: /config
+      containers:
+        - name: nginx-{{ .Values.name }}
+          image: nginx:latest
+          imagePullPolicy: "Always"
+          securityContext:
+            readOnlyRootFilesystem: true
+            allowPrivilegeEscalation: false
+          resources:
```
Comment (adamstrawson) [inline templates/deployment.yaml:44]: Worth making this a values block, just to future proof?

File: templates/istio-virtual-service.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{{- if eq .Values.ingress.enabled true }}
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: {{ .Values.name }}
+  namespace: dev
```
Comment (adamstrawson) [inline templates/istio-virtual-service.yaml:7]: Need to remove `namespace`

File: templates/pod-disruption-budget.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ .Values.name }}
+  namespace: dev
```
Comment (adamstrawson) [inline templates/pod-disruption-budget.yaml:5]: Namespace here too

File: values.yaml [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+deployment:
+  replicas: 1
+  container:
+    repository: europe-west2-docker.pkg.dev/platform-331118/sequence-web
+    tag: 1.0.0
+    port: "443"
+    env: {}
+ingress:
+  enabled: false
+hpa:
```
Comment (adamstrawson) [inline values.yaml:11]: This chart doesn't have HPA?

**Review by adamstrawson — CHANGES_REQUESTED** [review]

File: templates/deployment.yaml [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+          image: {{ .Values.deployment.container.repository }}/{{ .Values.name }}:{{ .Values.deployment.container.tag }}
+          imagePullPolicy: "Always"
+          command: ["cp", "-r", "/build/.", "/config"]
+          securityContext:
+            readOnlyRootFilesystem: true
+            allowPrivilegeEscalation: false
+          volumeMounts:
+            - name: config-volume
+              mountPath: /config
+      containers:
```
Comment (grrywlsn) [inline templates/deployment.yaml:37]: Let's add liveness + readiness just pointing to nginx:80

**Review by adamstrawson — APPROVED** [review]

**Pull request merged**